### PR TITLE
Add sample CLI tool

### PR DESCRIPTION
**Pull request by corneliusroemer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by corneliusroemer** [comment]
I've submitted a PR to the source repo to add a LICENSE.txt mentioning the license holder @hroptatyr. This seems to be the only thing missing.

**Comment by corneliusroemer** [comment]
LICENSE.txt was added to original repo, so this is ready for review
@conda-forge/staged-recipes please review if you have a minute

File: recipes/sample/build.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -x
+
+pwd
+
+./configure
```
Comment (wolfv) [inline recipes/sample/build.sh:7]: can you use `--prefix=${PREFIX}` and then `make install`?

Comment (corneliusroemer) [reply recipes/sample/build.sh:7]: I'm not quite sure where I should put the `--prefix=${PREFIX}`
After `make`, `make install`, `./configure`?
I tried `make install` but it didn't work so moved the binary manually.

Comment (wolfv) [reply recipes/sample/build.sh:7]: No, that shoudl be a argument to `./configure --prefix=${PREFIX}`

E.g. https://github.com/conda-forge/libidn11-feedstock/blob/6f8c64b6c42f04bc4f5235c990538fb37599f34e/recipe/build.sh#L42-L46 etc.

Comment (corneliusroemer) [reply recipes/sample/build.sh:7]: I see, thanks! Will try it. I've never done any c compilation stuff, this is all new to me.

Comment (corneliusroemer) [reply recipes/sample/build.sh]: Seems to have worked! Thanks @wolfv!

**Pull request merged**